src/app/components/payment-dialog/payment-dialog.component.spec.ts

```diff
@@ -13,11 +15,27 @@ import {
     PaymentDialogPayload
 } from './payment-dialog.component';
 import { TestProviders } from '../../../testing/test-providers';
+import { mockInput } from '../../../testing/interaction-helper';
 
 describe('PaymentDialogComponent', () => {
     let component: PaymentDialogComponent;
     let fixture: ComponentFixture<PaymentDialogComponent>;
 
+    function input(
```

Comment:
Question: Is this code used in other tests? If we reuse the same functionality in other tests, then it would be better to extract it to some test utils.

Reply:
Yes, there was one file using a similar code. I extracted them to the interaction-helper.

Reply:
Great thank you :)